prod.Dockerfile

```diff
@@ -1,40 +1,39 @@
-# FOSSology Dockerfile
```

Comment:
I actually went in with the opposite thought process, since for production deployment the command would be executed only once to set up while in a development environment the set up command might need to be executed multiple times for testing making it a bit cumbersome to execute a longer command both for hosting and pulling down docker containers. What are your thoughts on this?

Reply:
I agree with your comments but a person performing development using docker will have more expertise than a person simply cloning and setting up a server. Just to keep things simple keeping end user in mind, keeping the prod file as `Dockefile` and dev as `Dockerfile.dev` makes more sense to me.

Reply:
Agreed. I'll push an update soon.